src/CMakeLists.txt

```diff
@@ -72,13 +73,31 @@ set(BOT_SOURCES
     WorkerManager.cpp
     WorkerManager.h
     main.cpp
+    ${PROJECT_SOURCE_DIR}/CMakeLists.txt
+    ${PROJECT_SOURCE_DIR}/README.md
 )
 
 add_executable(CommandCenter ${BOT_SOURCES})
 
 # Show more warnings at compiletime.
 if (MSVC)
     target_compile_options(CommandCenter PRIVATE /W4 /EHsc)
+
+    set_target_properties( CommandCenter PROPERTIES
+        VS_DEBUGGER_WORKING_DIRECTORY "$<TARGET_FILE_DIR:CommandCenter>"
+        VS_DEBUGGER_COMMAND "$<TARGET_FILE:CommandCenter>"
+        VS_DEBUGGER_COMMAND_ARGUMENTS "Ladder2019Season3/AcropolisLE.SC2Map"
+    )
+
+    # Get Readme and CMakelists to show up in Solution Explorer
+    set(OTHER
```

Comment:
This is a bit inconvenient to have all CMakeLists.txts in single folder as cmake files have strong connection with the folder (code of the folder) they are located. If you wish to add CMakeLists.txt into source code tree, I would recommend to reflect the original project structure and avoid 'other' groups.